spec/models/list_spec.rb

```diff
@@ -11,6 +11,10 @@
   it 'has a valid factory' do
     expect(@list).to be_valid
   end
+  it 'sends an email on creation' do
+    last_email = ActionMailer::Base.deliveries.last
+    expect(last_email.subject).to include("Congratulations for creating a wishlist!")
```

Comment:
this will break if the `before :each` block sends an email for any other reason - consider creating a list in the test body instead